### PR TITLE
Display Trainee, Supervisor

### PR DESCRIPTION
**Pull request by SSalekin**

### CODE REVIEW

File: app/views/supervisor/courses/show.html.erb [hunk 1]
```diff
@@ -13,12 +13,94 @@
...
+        <font size="3">
+          <%= link_to t(:add_edit_enrollment),
+            supervisor_course_course_users_path(@course),
+              class: "btn btn-success btn-xs"%>
+        </font>
+      </p>
+      <br>
+      <p>
+        <font size="3">
+          <%= t 'supervisor' %>: (<%= @course.users.supervisors.count %>)
```
Comment (chuanhtuan) [inline app/views/supervisor/courses/show.html.erb]: @course.supervisors.count

File: app/views/supervisor/courses/show.html.erb [hunk 2]
```diff
@@ -13,12 +13,94 @@
...
+        </font>
+        <button id="btnShowSupervisor" type="button"
+          class="btn btn-default btn-xs" data-toggle="modal"
+            data-target="#divShowSupervisor">
+            <%= t 'view' %>
+        </button>
+       </p>
+      <p>
+        <font size="3">
+          <%= t 'trainee' %>: (<%= @course.users.trainees.count %>)
```
Comment (chuanhtuan) [inline app/views/supervisor/courses/show.html.erb]: @course.trainees.count

File: app/views/supervisor/courses/show.html.erb (hunk 1 above)
Comment (SSalekin) [inline app/views/supervisor/courses/show.html.erb]: we have relation between Course and User.
'Supervisor' is the role of User. So we can't access the number through this code 
'@course.supervisors.count'

File: app/views/supervisor/courses/show.html.erb (hunk 2 above)
Comment (SSalekin) [inline app/views/supervisor/courses/show.html.erb]: we have relation between Course and User.
'Trainee' is the role of User. So we can't access the number through this code 
'@course.trainees.count'

File: app/models/course.rb [hunk 3]
```diff
@@ -1,6 +1,9 @@
 class Course < ActiveRecord::Base
   has_many :course_users, dependent: :destroy
   has_many :users, through: :course_users
+  has_many :trainees, -> {trainees}, class_name: "User", through: :course_users
```
Comment (chuanhtuan) [inline app/models/course.rb]: you can use where condition of has_many

**Pull request merged**